src/world.rs

```diff
@@ -420,22 +409,29 @@ impl<C> LazyUpdateInternal for LazyRemovalInfo<C>
     where C: Component + Send + Sync
 {
     fn update(self: Box<Self>, world: &World) {
-        world.write::<C>().remove(self.entity);
+        world.write::<C>().remove(self.0);
     }
 }
 
-/// Lazy updates can be used after creating a new
-/// entity in a system. This way, none of the actual
-/// component storages have to be borrowed mutably.
-///
+impl LazyUpdateInternal for Box<Fn(&World) + Send + Sync> {
```

Comment:
Not sure, I tried without that Box first, but ran into problems. The compiler suggests to add a 'static bound to F in execute, but the passed in closures apparently don't have 'static lifetime. Any idea how to solve this?